database/migrations/1733672331665_create_student_organisations_table.ts

```diff
@@ -0,0 +1,44 @@
+import { BaseSchema } from '@adonisjs/lucid/schema'
+
+export default class extends BaseSchema {
+  protected tableName = 'student_organisations'
+  protected sources = ['student_department', 'manual', 'pwr_active']
+  protected organization_type = [
```

Comment:
to keep everything neat and tidy, you should define your enums in `/app/enums` and import them here

Reply:
we've previously talked about how this wasn't a great idea if we ever decide to add enum values, as doing so will modify the behavior of already merged migration files without a direct change to them that would be visible in the git logs.

Reply:
okay, so just to get on the same page - you suggest to define enum used in models in `/app/enums` and the ones for migrations directly in the migration files?

Reply:
exactly, this way you can't possibly change the behavior of a migration file without editing it directly.

Reply:
alright, just keep reminding me if i keep mixing this up